src/foam/nanos/http/ClassesWebAgent.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.http;
+
+import foam.core.*;
+import foam.dao.DAO;
+import java.net.URL;
+import java.net.URLClassLoader;
+import foam.nanos.boot.NSpec;
+import java.io.PrintWriter;
+
+public class ClassesWebAgent
```

Comment:
This doesn't match the description of the PR.

Reply:
that was my mistake i was in the wrong branch when putting it will update the description

Reply:
fixed